lib/orogen/templates/tasks/tasks.pc

```diff
@@ -32,5 +32,5 @@ Libs: <%= libs.to_a.sort.join(" ") %> -L${libdir} -l<%= project.name %>-tasks-<%
 <% if project.extended_state_support? %>
 <%   cflags << "-I${includedir}/#{project.name}/types" %>
 <% end %>
-Cflags: -I${includedir} <%= cflags.to_a.sort.join(" ") %>
+Cflags: -I${includedir} <%= cflags.to_a.join(" ").split(" ").uniq.sort.join(" ") %>
```

Comment:
When I look at the content of cflags, e.g., for tools/logger I do encounter the following:
(change in orogen: lib/orogen/templates/tasks/tasks.pc to contain # Cflags: <%= cflags.to_a %>

```
Cflags: ["-I/opt/workspace/rock/dev/install/include -DOROCOS_TARGET=gnulinux -I/opt/workspace/rock/dev/install/include/orocos -I/opt/workspace/rock/dev/install/include/orocos/std/types", "-I/opt/workspace/rock/dev/install/include -DOROCOS_TARGET=gnulinux -I/opt/workspace/rock/dev/install/include/base/backward -I/usr/include/eigen3 -I/opt/workspace/rock/dev/install/include/orocos -I/opt/workspace/rock/dev/install/include/orocos/std/types -I/opt/workspace/rock/dev/install/include/orocos/base/types", "-I${includedir}/logger/types"]